Resources/Prototypes/SS220/Entities/Mobs/NPCs/animals.yml

```
@@ -0,0 +1,107 @@
+- type: entity
+  name: Охранная собака
+  parent: SimpleMobBase
+  id: MobSecDog
+  description: Как ни странно, эта свинья на самом деле собака.
+  components:
+  - type: GhostRole
+    makeSentient: true
+    allowSpeech: true
+    allowMovement: true
+    name: "Собака Службы Безопасности"
```

```suggestion
    name: "собака Службы Безопасности"
```
Тоже не имя собственное